aimmo_runner/minikube.py

```diff
@@ -2,82 +2,13 @@
 from __future__ import print_function
 
 import docker
-import errno
 import kubernetes
 import os
 import re
 import socket
-import stat
 import yaml
 import platform
-from run import run_command
-from urllib import urlretrieve
-from urllib2 import urlopen
-
-BASE_DIR = os.path.abspath(os.path.dirname(__file__))
-TEST_BIN = os.path.join(BASE_DIR, 'test-bin')
-OS = platform.system().lower()
-FILE_SUFFIX = '.exe' if OS == 'windows' else ''
-KUBECTL = os.path.join(TEST_BIN, 'kubectl%s' % FILE_SUFFIX)
-MINIKUBE = os.path.join(TEST_BIN, 'minikube%s' % FILE_SUFFIX)
-
-
-def create_test_bin():
-    try:
-        os.makedirs(TEST_BIN)
-    except OSError as err:
-        if err.errno != errno.EEXIST:
-            raise
-
-
-def get_latest_github_version(repo):
-    result = urlopen('https://github.com/%s/releases/latest' % repo)
-    return result.geturl().split('/')[-1]
-
-
-def download_exec(url, dest):
-    dest = urlretrieve(url, dest)[0]
-    make_exec(dest)
-
-
-def make_exec(file):
-    current_stat = os.stat(file)
-    os.chmod(file, current_stat.st_mode | stat.S_IEXEC)
-
-
-def binary_exists(filename):
-    # Check if binary is callable on our path
-    try:
-        run_command([filename], True)
-        return True
-    except OSError:
-        return False
-
-
-def download_kubectl():
```

Comment:
we should change the readme to signify that you have to download and install kubectl and minikube and a vm thing yourself

Reply:
Done 👍